tests/ui/widgets/Text.cxx

```diff
@@ -0,0 +1,45 @@
+#include <catch2/catch.hpp>
+#include "../../src/engine/ui/widgets/Text.hxx"
+#include "../../src/engine/basics/Settings.hxx"
+
+using string = std::string;
+
+TEST_CASE("I can create a Text widget", "[ui][widgets][text]")
+{
+  Text *text = new Text;
+  CHECK(text != nullptr);
+}
+
+TEST_CASE("I can set text on a Text widget", "[ui][widgets][text]")
+{
+  Text text;
+  text.setText("Test");
+  CHECK(text.getUiElementData().text == "Test");
+}
+
+TEST_CASE("I can change the font size of a Text widget", "[ui][widgets][text]")
+{
+  Text text;
+  text.setFontSize(30);
+  text.setText("Test");
+  text.setFontSize(100);
+  CHECK(text.getUiElementData().text == "Test");
+  CHECK(text.getFontSize() == 100);
+}
+
+TEST_CASE("I can draw a Text widget", "[ui][widgets][text]")
+{
+  Text *text = new Text;
+  text->draw();
+  text->setText("This is a test");
+  text->draw();
+  CHECK(text != nullptr);
```

Comment:
Ideally, here you would take a screenshot or verify that it actually drew the text. Unfortunately you cant. Maybe for now just `CHECK_NOTHROW(text->draw());` and add a `@todo actually verify it draws the text`

Reply:
done that :) the enhancement with the screenshot is a good idea once we've set up xvfb